app/routes/admin/users/list.js

```diff
@@ -13,91 +13,29 @@ export default Route.extend({
   },
   model(params) {
     this.set('params', params);
-    return [{
-      name        : 'Test name 1',
-      email       : 'Testname1@gmail.com',
-      status      : 'active',
-      systemRoles : ['unverified user1', 'unverified user2'],
-      eventRoles  : [
+    let filterOptions = [];
+    if (params.users_status === 'active') {
+      filterOptions = [
         {
-          name  : 'Organizer',
-          event : {
-            name: 'Event One'
-          }
-        },
-        {
-          name  : 'Organizer',
-          event : {
-            name: 'Event Two'
-          }
-        }
-      ],
-      createdAt      : new Date(),
-      lastAccessedAt : new Date()
-    },
-    {
-      name        : 'Test name 2',
-      email       : 'Testname2@gmail.com',
-      status      : 'inactive',
-      systemRoles : ['unverified user1', 'unverified user2'],
-      eventRoles  : [
-        {
-          name  : 'Organizer',
-          event : {
-            name: 'Event two'
-          }
-        }
-      ],
-      createdAt      : new Date(),
-      lastAccessedAt : new Date()
-    },
-    {
-      name        : 'Test name 3',
-      email       : 'Testname3@gmail.com',
-      status      : 'active',
-      systemRoles : ['unverified user1', 'unverified user2'],
-      eventRoles  : [
-        {
-          name  : 'Organizer',
-          event : {
-            name: 'Event One'
-          }
-        }
-      ],
-      createdAt      : new Date(),
-      lastAccessedAt : new Date()
-    },
-    {
-      name        : 'Test name 4',
-      email       : 'Testname4@gmail.com',
-      status      : 'active',
-      systemRoles : ['unverified user1', 'unverified user2'],
-      eventRoles  : [
-        {
-          name  : 'Organizer',
-          event : {
-            name: 'Event One'
-          }
+          name : 'state',
```

Comment:
@geekyd yes I am aware of that. I asked niranjan about how to sort the users but he was not well so @fossasia/open-event-frontend can anyone help how to sort the users based on active and deleted and also tell the status whether they are active or inactive?